packages/@react-spectrum/menu/src/MenuItem.tsx

```diff
@@ -110,12 +133,15 @@ export function MenuItem<T>(props: MenuItemProps<T>) {
                 <CheckmarkMedium
                   slot="checkmark"
                   UNSAFE_className={
-                        classNames(
-                          styles,
-                          'spectrum-Menu-checkmark'
-                        )
-                      } />
-                  }
+                    classNames(
+                      styles,
+                      'spectrum-Menu-checkmark'
+                    )
+                  } />
+              }
+              {
+                isUnavailable && <InfoOutline slot="end" aria-label={stringFormatter.format('unavailable')} />
```

Comment:
Check the designs here https://spectrum.adobe.com/page/menu/#Unavailable Looks like there should be a bit more padding and slightly different color

Reply:
color is the same as in the XD file, (34, 34, 34) in light mode